rosette/src/Reader.cc

```diff
@@ -1352,20 +1352,21 @@ Ob* Reader::resumeExpr() {
         int c = fgetc(my->file);
 #endif
 
-        if (c == EOF && RBL_WOULDBLOCK)
+        if (EOF == c && EWOULDBLOCK == errno) {
```

Comment:
Can we have some parens here?

Reply:
Parents aren't really necessary if you put constants on the left. Do you feel they would improve readability? I'm ambivalent, but generally prefer to use the unambiguous syntax that includes the fewest tokens.

Reply:
I got tired of looking up operator precedence years ago, and just prefer to see parentheses. I don't feel strongly enough about it to make you change it though.